tsdb/blockwriter.go

```diff
@@ -105,12 +105,12 @@ func (w *BlockWriter) Flush(ctx context.Context) (ulid.ULID, error) {
 	if err != nil {
 		return ulid.ULID{}, fmt.Errorf("create leveled compactor: %w", err)
 	}
-	id, err := compactor.Write(w.destinationDir, w.head, mint, maxt, nil)
+	ids, err := compactor.Write(w.destinationDir, w.head, mint, maxt, nil)
 	if err != nil {
 		return ulid.ULID{}, fmt.Errorf("compactor write: %w", err)
 	}
 
-	return id, nil
+	return ids[0], nil
```

Comment:
Should we also return the list of block ids here?

Reply:
I understand that this block writer is only used by promtool cli to backfill blocks from rules and openmetrics file.

It is not used by TSDB itself other than in tests.